jsonparse.ps1

```diff
@@ -0,0 +1,83 @@
+$location=$args[0]
+$repo=$args[1]
+$name=$args[2]
+
+$location
+$repo
+$name
+$repo.Title
+$JSON = (Get-Content $location) -join "`n" | ConvertFrom-Json
+
+$base = "https://github.com/"
+if($repo.Title -eq "github")
+{
+  $base = "https://github.com/"
+}
+#Renaming the key value of file
+$URI = $JSON.runs
+$JSON.runs.files.PSObject.Properties | foreach-object {
+  $URIReplacement = $_.Name
```

Comment:
This apparently works, but it is a dangerous coding pattern. The problem is that you're modifying a collection _while you are iterating over it_. In C#, [you'd get an exception](http://stackoverflow.com/questions/16497028/how-add-or-remove-object-while-iterating-collection-in-c-sharp).

Now for all I know this is perfectly safe in PowerShell, but it sets off alarm bells. I suggest you reseach this to find out if it's a safe pattern in PowerShell.